squarespace-template.js

```diff
@@ -795,51 +797,34 @@ compilePages = function ( cb ) {
 compileStylesheets = function ( cb ) {
     siteCss = "";
 
-    var styles = [{
-        name: "reset.css",
-        path: path.join( directories.styles, "reset.css" )
-    }];
-
+    styles = [
+        '@import "reset.css";'
+    ];
+    
     for ( var i = 0, len = config.template.stylesheets.length; i < len; i++ ) {
-        styles.push({
-            name: config.template.stylesheets[ i ],
-            path: path.join( directories.styles, config.template.stylesheets[ i ] )
-        });
+        styles.push(
+            '@import "' + config.template.stylesheets[ i ] + '";'
+        );
     }
 
-    function read() {
-        if ( !styles.length ) {
-            cb();
-
-        } else {
-            var style = styles.pop();
-
-            sqsUtil.isFile( style.path, function ( exists ) {
-                if ( !exists ) {
-                    read();
+    fs.writeFile( path.join(directories.styles, 'style.less') , styles.join("\n") , function(err) {
+        if(err) {
+            return sqsLogger.log( "error", err);
+        }
 
+        execFile(
+            "/usr/local/bin/sqs-lessc",
+            ['style.less', 'style.css'],
```

Comment:
oh, also, this will render to file, so I dunno how well that will play with livereload stuff.